go/libkb/globals.go

```diff
@@ -63,7 +63,7 @@ type GlobalContext struct {
 
 func NewGlobalContext() *GlobalContext {
 	return &GlobalContext{
-		Log: logger.New("keybase"),
+		Log: logger.New("keybase", OutputWriter()),
```

Comment:
Logging to standard output will break all our interactive commands, since it will Gum up standard output with log info.